middleware/app/Http/Controllers/Api/V1/ProviderPlatformController.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Http\Controllers\api\V1;
+
+use App\Models\ProviderPlatform;
+use Illuminate\Http\Request;
+use ProviderPlatformServices;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\StoreProviderPlatformRequest;
+use App\Http\Resources\ProviderPlatformResource;
+
+class ProviderPlatformController extends Controller
+{
+
+    // List all provider platforms known to the system
+    // ****************************************************
+    // GET: /api/provider_platforms/
+    // ****************************************************
+    public function index()
+    {
+        return ProviderPlatformResource::collection(ProviderPlatform::all(['*']));
+    }
+    //
+    // List information on a specific provider platform
+    // ****************************************************
+    // GET: /api/provider_platforms/{request_body}
+    // @param Request $request
+    // @return JsonResponse
+    // ****************************************************
+    public function show(Request $request, $id): ProviderPlatformResource|\Illuminate\Http\JsonResponse
+    {
+        $providerPlatform = ProviderPlatform::where('id', $id)->first();
+        if ($providerPlatform) {
+            return new ProviderPlatformResource($providerPlatform);
+        } else {
+            return response()->json(['error' => 'Invalid request body'], 401);
+        }
+    }
+    // Create a new provider platform (does not register connection)
+    // ****************************************************
+    //  POST: /api/provider_platform/{request_body}
+    //  Request $request
+    // ****************************************************
+    public function store(StoreProviderPlatformRequest $req): ProviderPlatformResource
+    {
+        try {
+            $validated = $req->validated();
+        } catch (\Exception) {
+            return response()->json(['error' => 'Invalid request body'], 401);
+        }
+        $provider = ProviderPlatform::create($validated);
+        // Create a new platform connection.
+        return response()->json($provider);
```

Comment:
@PThorpe92 does this match the `ProviderPlatformResource` return type indicated in the signature? I mean the Resource class does return a JSON array, but idk if the strictness would throw something...